### DISCUSSION TITLE
Ender3 S1 Bed leveling

### DISCUSSION THREAD

**Discussion by pico45** [mriscoc/Ender3V2S1#773] [Q&A]
i use in my startup code
G29
M420 S1
that all works too.
But when leveling is finished and printing starts, the display is still in bed leveling mode and is waiting for continue or save.
is there a command where the display automatically exits the bed leveling mode without having to press the button ????

**Comment by mriscoc** [comment]
In a new line write C108 after the G29

**Comment by pico45** [comment]
thanks MRISCOC that worked.
